src/library.js

```diff
@@ -980,17 +980,23 @@ LibraryManager.library = {
     if (_tzset.called) return;
     _tzset.called = true;
 
+    var currentYear = new Date().getFullYear();
+    var winter = new Date(currentYear, 0, 1);
+    var summer = new Date(currentYear, 6, 1);
+    var winterOffset = winter.getTimezoneOffset();
+    var summerOffset = summer.getTimezoneOffset();
+    // Local standard timezone offset. Local standard time is not adjusted for daylight savings.
+    // Using Math.max we can get retrieve the local standard time for the timezone.
```

Comment:
```suggestion
    // Daylight savings adjusts the time to a lower value, so take the maximum here.
```
Is this comment right? I am inferring this from you doing the maximum, I don't actually understand time zones...

Reply:
Yup you're right, changed it hopefully it's a little more clear now.